docs/docs-content/clusters/edge/site-deployment/initial-setup.md

```diff
@@ -0,0 +1,53 @@
+---
+sidebar_label: "Initial Edge Host Configuration"
+title: "Initial Edge Host Configuration"
+description: "Instructions for configuring the Edge host when booting up the Edge host for the first time."
+hide_table_of_contents: false
+sidebar_position: 32
+tags: ["edge"]
+---
+
+When you boot up an Edge host for the first time after installation, you will be prompted to enter several
+configurations for the Edge host and its network environment. You may already have specified some of these
```

Comment:
```suggestion
When you boot up an Edge host for the first time after installation, you will be prompted to configure the Edge host and its network environment. You may already have specified some of these
```

Reply:
I would simplify the language here and make it more action-oriented: you're prompted to enter a configuration vs. you're prompted to configure